docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Check working directory is the root directory of the PDKGO repository
+if [ ! -d "docs/md" ]; then
+    echo 'Please run this script from the root directory of the PDKGO project.'
+    exit 1
+fi
+# Check hugo extended is installed
+if ! type "hugo" > /dev/null || ! hugo version | grep -qG "\bextended\b"; then
```

Comment:
You might be able to simplify this to:
```suggestion
if ! hugo version | grep -q 'extended'; then
```